### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java [hunk 1]
```diff
@@ -56,7 +54,10 @@
   public static final String CLIENT_APIS_GROUP = "clientapis"; //$NON-NLS-1$
 
   private static final Logger logger = Logger.getLogger(CloudLibraries.class.getName());
-  private static final ImmutableMap<String, Library> libraries = loadLibraryDefinitions();
+
+  // Mostly immutable info of libraries loaded from plugin.xml, but LibraryFile versions can be
+  // updated dynamically, e.g., to latest available release versions.
+  private static final Map<String, Library> libraries = loadLibraryDefinitions();
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java]: Why did you remove the ImmutableMap type? That seemed useful.
However we probably shouldn't comment it as "Moslty immutable".

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java [hunk 2]
```diff
@@ -132,24 +133,22 @@ public static Library getLibrary(String id) {
     }
   }
   
-  private static ImmutableMap<String, Library> loadLibraryDefinitions() {
+  private static Map<String, Library> loadLibraryDefinitions() {
     IConfigurationElement[] elements = RegistryFactory.getRegistry().getConfigurationElementsFor(
         "com.google.cloud.tools.eclipse.appengine.libraries"); //$NON-NLS-1$
-    ImmutableMap.Builder<String, Library> builder = ImmutableMap.builder();
+    Map<String, Library> map = new HashMap<>();
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java]: I don't concur with the change to a mutable map here.

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java [hunk 3]
```diff
@@ -65,6 +65,7 @@ public void importAppEngineStandardJava7_from1_3_1() throws IOException, CoreExc
 
     updateOldContainers();
 
+    ProjectUtils.waitForProjects(project);
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java]: This looks significant enough to go into a focused PR of its own rather than a PR of assorted minor cleanups.

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java (hunk 1 above)
Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java]: OK, I'll revert it.

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java (hunk 3 above)
Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java]: Backing out and created #2512.
